cypress/fixtures/docs/staticJSObjects.md

```diff
@@ -0,0 +1,139 @@
+# Table of contents: 
+[JS Objects](#js-objects)
+ - [Firebolt Calls](#firebolt-calls)
+   - [How to Configure Firebolt Calls](#how-to-configure-firebolt-calls)
+   - [Example 1](#example-1)
+   - [Example 2](#example-2)
+ - [Validation Objects](#validation-objects)
+   - [How to Configure Validation Objects](#how-to-configure-validation-objects)
+   - [Example 1](#example-1-1)
+   - [Example 2](#example-2-1)
+- [Usage](#usage)
+- [Advanced Support](#advanced-support)
+
+## JS Objects
```

Comment:
Think this should be doc title and can remove from TOC. Docs should have Title, brief description, TOC.

Reply:
addressed the comments